scripts/vars.sh

```diff
@@ -3,79 +3,97 @@
 set -eu
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-DEPENDENCIES="jq bats"
-
-# check and install dependencies
-echo "\nChecking dependencies... "
-deps=0
-for name in $DEPENDENCIES
-do
-    [[ $(type $name 2>/dev/null) ]] || { echo "\n    * $name is required to run this script;";deps=1; }
-done
-[[ $deps -ne 1 ]] && echo "OK\n" || { echo "\nInstall the missing dependencies and rerun this script...\n"; exit 1; }
-
-# define STRIDE vars
 STATE=$SCRIPT_DIR/state
-PORT_ID=26656  # 36564 
-MAIN_ID=0
-BLOCK_TIME='5s'
+PEER_PORT=26656
+
+# DENOMS
+IBC_STRD_DENOM='ibc/FF6C2E86490C1C4FBBD24F55032831D2415B9D7882F85C3CC9C2401D79362BEA'
+IBC_ATOM_DENOM='ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
+ATOM_DENOM='uatom'
+STRD_DENOM='ustrd'
+STATOM_DENOM="stuatom"
 
-STRIDE_CHAIN=STRIDE
-STRIDE_NODE_NAMES=(stride1 stride2 stride3)
-STRIDE_MAIN_NODE=${STRIDE_NODE_NAMES[$MAIN_ID]}
+BLOCK_TIME='5s'
+DAY_EPOCH_INDEX=1
+DAY_EPOCH_DURATION="100s"
+STRIDE_EPOCH_INDEX=2
+STRIDE_EPOCH_DURATION="40s"
+UNBONDING_TIME="200s"
+MAX_DEPOSIT_PERIOD="30s"
+VOTING_PERIOD="30s"
+
+VAL_TOKENS=5000000000000
+STAKE_TOKENS=3000000000000
```

Comment:
nice, this will keep the RR increases in check. Might even suggest raising it to ~90% of VAL_TOKENS

Reply:
done